src/lib/analytics/types.ts

```diff
@@ -32,6 +32,8 @@ export type TrackPropertiesMap = {
   'Post:ThreadMute': {} // CAN BE SERVER
   'Post:ThreadUnmute': {} // CAN BE SERVER
   'Post:Reply': {} // CAN BE SERVER
+  'Post:EditThreadgateOpened': {}
```

Comment:
these are old mixpanel events, intentional? time to delete these maybe.

Reply:
Oh -- I'm unsure, do we not still use `track()` ? That API depends on the declarations in `TrackPropertiesMap`

Reply:
yeah `track` is the old one, `logEvent` is the new one. we can probably just rename/remove at this point